### DISCUSSION TITLE
Possible to display all results from a List<string>?

### DISCUSSION THREAD

**Discussion by mrlucmorin** [waf/CSharpRepl#376] [Q&A]
Hi,

At the moment, when I display elements from a List<string>, only the first 15 elements are displayed. Is there a setting to instruct the REPL to display ALL elements of the list?

Thanks

**Comment by kindermannhubert** [comment]
Currently, it's not configurable. However, it's dynamic and depends on console window height.
For simple output (default shortcut is `Enter`) we output `0.5 * WindowRows` items.
For detailed output (default shortcut is `Ctrl+Enter`)we output `2 * WindowRows` items.

See: https://github.com/waf/CSharpRepl/blob/958554ff490ec74d86ea2df357a67e1b3af92dc7/CSharpRepl.Services/Roslyn/Formatting/LengthLimiting.cs#L31-L32

The size limiting is necessary because collections could be large or potentially infinite (custom implementation of `IEnumerable<T>`).

**Comment by mrlucmorin** [comment]
Hi,

Thanks for the reply. Is this something you're already working on (making it available as a setting or config file), or should I create an issue?

Yeah, IEnumerable can be tricky if you're not careful 😁

Keep up the excellent work on REPL, I love it!!!

Cheers

**Comment by kindermannhubert** [comment]
Not working on.
Creating issue would be fine. Thanks.
